noir-projects/aztec-nr/aztec/src/state_vars/private_set.nr

```diff
@@ -44,35 +44,41 @@ impl<Note> PrivateSet<Note> {
     // DEPRECATED
     fn assert_contains_and_remove(_self: Self, _note: &mut Note, _nonce: Field) {
         assert(
-            false, "`assert_contains_and_remove` has been deprecated. Please call PXE.addNote() to add a note to the database. Then use PrivateSet.get_notes() and PrivateSet.remove() in your contract to verify and remove a note."
+            false, "`assert_contains_and_remove` has been deprecated. Please call PXE.addNote() to add a note to the database. Then use PrivateSet.get_notes() in your contract to verify and remove a note."
         );
     }
 
     // DEPRECATED
```

Comment:
Should we not get rid of this one if deprecated, why keep it? 👀  better fit for separate issue, just saw it here.